### PR TITLE
KAFKA-6729: Follow up; disable logging for source KTable.

### PR DESCRIPTION
**Pull request by guozhangwang**
### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by guozhangwang** [comment]
@mjsax @vvcephei @bbejeck

File: streams/src/test/java/org/apache/kafka/streams/StreamsBuilderTest.java [hunk 1]
```diff
@@ -279,7 +279,11 @@ public void shouldReuseSourceTopicAsChangelogs() {
 
         final InternalTopologyBuilder internalTopologyBuilder = TopologyWrapper.getInternalTopologyBuilder(builder.build());
 
-        assertThat(internalTopologyBuilder.topicGroups().get(0).stateChangelogTopics.keySet(), equalTo(Collections.singleton("topic")));
```
Comment (mjsax) [inline streams/src/test/java/org/apache/kafka/streams/StreamsBuilderTest.java:282]: This line should have told us that the original PR was not sufficient for a fix... :(

**Review by mjsax — APPROVED** [review]
LGTM.

**Comment by guozhangwang** [comment]
retest this please

**Comment by mjsax** [comment]
@guozhangwang `StreamsPartitionAssignorTest.shouldGenerateTasksForAllCreatedPartitions` fails.

**Comment by guozhangwang** [comment]
@mjsax just pushed another commit.

**Comment by mjsax** [comment]
Retest this please.

**Pull request merged**